rapid7/metasploit-framework
# Xerox Admin password extractor.

This PR is for the Praedasploit module that extracts the admin password for the web administrator interface on Xerox printers. 

Below is an example of the options available in the module.
![xerox-options](https://cloud.githubusercontent.com/assets/3900108/4802106/1eafba0e-5e3e-11e4-9e78-57ba7cb6802d.png)

Testing Steps:
set JPORT to the JetDirect port on the remote Xerox printer
Set RHOST to the IP address of the remote Xerox printer
Run the module and see the magic happen.

Below is an example of the output when the module is run. 
![console_pwd_enum-working](https://cloud.githubusercontent.com/assets/3900108/4802235/800b4c62-5e40-11e4-8f0e-9236ea34d1db.png)

Please let me know if you have any questions.
